zyte_common_items/pages/serp.py

```diff
@@ -0,0 +1,40 @@
+from typing import List, Optional
+
+import attrs
+from web_poet import Returns
+
+from zyte_common_items.components import SerpOrganicResult
+from zyte_common_items.fields import auto_field
+from zyte_common_items.items import Serp, SerpMetadata
+
+from .base import BasePage, Page
+from .mixins import HasMetadata
+
+
+class BaseSerpPage(BasePage, Returns[Serp], HasMetadata[SerpMetadata]):
+    pass
+
+
+class SerpPage(Page, Returns[Serp], HasMetadata[SerpMetadata]):
```

Comment:
Just wondering, how and where will `SerpPage` be used?

Reply:
It should work as a base class for custom page objects for Serp items.